src/test/java/seedu/address/logic/parser/DeleteTaskCommandParserTest.java

```diff
@@ -25,4 +25,23 @@ public void parse_invalidArgs_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse("uid/1"));
         assertThrows(ParseException.class, () -> parser.parse(""));
     }
+
+    @Test
+    public void parse_invalidUid_throwsParseException() {
+        try {
+            parser.parse("ui/ 1");
+        } catch (ParseException e) {
+            assertEquals(e.getMessage(), "Invalid UID format! Correct format: uid/<UID>");
```

Comment:
I think we should keep it consistent with our UG. "uid/UID" instead of "uid/<UID>"